src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -87,6 +88,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String user = userService.getCurrentUser().getEmail();
     String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
+    //Obtain and set Keys
+    // String parentKey = marker.getKey();
```

Comment:
I am a bit unsure of this part. How do I get the market key and add to message? Can someone please help me understand this a little more?